modules/modules.test.js

```diff
@@ -18,4 +18,10 @@ describe("Test if morse code translates to english correctly", () => {
       ".... . .---- .---- ---   .-...   -... ...-- -.--"
     );
   });
+  test("Outputs any symbol not in the code dictionary as it is typed", () => {
+    expect(convertToMorse("///")).toBe("/ / /");
+  });
+  test("Morse converts to english and joins the letters to form a word", () => {
+    expect(convertToEnglish("... --- ...")).toBe("sos");
+  });
 });
```

Comment:
Add Tests to make sure that sentences get translated back to English properly. Also check that special char are translated in English correctly

Reply:
Added two new test cases, one for "hello world" and another to handle symbols, changed the function so when two spaces are entered a new word begins